activestorage/test/service/gcs_public_service_test.rb

```diff
@@ -9,6 +9,10 @@ class ActiveStorage::Service::GCSPublicServiceTest < ActiveSupport::TestCase
 
     include ActiveStorage::Service::SharedServiceTests
 
+    test "public acl options" do
+      assert_equal "public_read", @service.upload_options[:acl]
+    end
```

Comment:
Can we test the desired effect(s) of setting this option automatically rather than just testing that we set it?

Reply:
I was following the lead of the `S3PublicServiceTest` but yep, I'll extend!

Reply:
Done, let me know what you think :) The existing test already covers that the URL is generated correctly for a public upload so with the new on I tested that the ACL includes the `allUsers` group which seems to be Google's way of saying "Internet!" based off what I could read in https://cloud.google.com/storage/docs/access-control/lists